### DISCUSSION TITLE
Wrapping Crystals

### DISCUSSION THREAD

**Discussion by jrellin** [OpenGammaProject/Open-Gamma-Detector#48] [Q&A] [answered]
Just finished my first board and was wondering if wrapping crystals with something like Teflon tape would work or would they saturate the sensor. I will be using some 3 x 3 x 20 mm LYSO crystals I have. Might also have BGO somewhere.

**Comment by NuclearPhoenixx** [comment, marked as answer]
If you're using a crystal that's not fully enclosed like an NaI scintillator in an aluminium case, I'd always recommend to wrap it either in PTFE tape or just aluminium foil to reflect as much light back as possible. The sides of the crystal should also be polished for the best results.

Keep in mind that this does not light-proof the crystal (especially the PTFE tape). You still need to wrap it in a couple layers of black tape or something so that no ambient light gets into the assembly.

Hope that helps.
